### PR TITLE
KZOO-346: update the kazoo-core services quantity handlers to emit quantities for chat.getstream.enabled

### PR DESCRIPTION
**Pull request by Navoda626**
KZOO-346: update the kazoo-core services quantity handlers to emit quantities for chat.getstream.enabled

Charge $1 per user when the getstream chat workspace is enabled (user document will have chat.getstream.enable=true) for a user by an Admin

### CODE REVIEW

**Comment by nkarunarathne** [comment]
Changes look good

File: kazoo_services/src/kz_services_quantities.erl [hunk 1]
```diff
@@ -650,6 +651,13 @@ is_first_transition_to_state(JObj) ->
                                         ),
     length(StateTransitions) =:= 1.
 
+-spec calculate_chat_updates(kz_json:object(), kz_term:proplist()) -> kz_term:proplist().
+calculate_chat_updates(JObj, Updates) ->
+    case kz_json:is_true([<<"chat">>, <<"getstream">>, <<"enabled">>], JObj, 'false') of
+        'true' -> [{[<<"chat">>, <<"getstream">>], 1} | Updates];
```
Comment (jamesaimonetti) [inline kazoo_services/src/kz_services_quantities.erl:661]: Is the `1` for one dollar? or for `1` quantity?

Comment (Navoda626) [reply kazoo_services/src/kz_services_quantities.erl:661]: As I understand it is 1 quantity

Comment (Navoda626) [reply kazoo_services/src/kz_services_quantities.erl:661]: But in the story of the ticket it said $1 charge per user. https://2600hz-commercial.atlassian.net/browse/MUDA-13

So I'm not sure how this quantity convert into $. I will check on that

Comment (Navoda626) [reply kazoo_services/src/kz_services_quantities.erl:661]: This 1 is for 1 quantity and conversion to currency is done in kz_currency module

Comment (jamesaimonetti) [reply kazoo_services/src/kz_services_quantities.erl:661]: update here, right? should be pvt_chat now, yes?

Comment (Navoda626) [reply kazoo_services/src/kz_services_quantities.erl:661]: Yes. The user document field is updated

**Pull request merged**